scripts/trim_sdk.py

```diff
@@ -108,7 +111,9 @@ def _print_folder_size(folder):
 
 
 def _get_all_sdks_to_trim():
-    resource_types = [k for k, v in AZURE_API_PROFILES['latest'].items() if k.import_prefix.startswith('azure.mgmt')]
+    # azure.mgmt.network has been removed in https://github.com/Azure/azure-cli/pull/25451
+    resource_types = [k for k, v in AZURE_API_PROFILES['latest'].items() if k.import_prefix.startswith('azure.mgmt')
+                      and k.import_prefix != 'azure.mgmt.network']
```

Comment:
`azure.mgmt.network` is removed from dependency in #25451

Reply:
I don't think we should hard-code `azure.mgmt.network` here. What if another SDK is removed, such as `azure.mgmt.compute`? It won't be a good idea to keep modifying this `scripts/trim_sdk.py`.